Automattic/wp-calypso
# Update copy for Backup product cards

### Changes proposed in this Pull Request

This PR includes the final copy for the Jetpack Backup, Backup _Daily_, and Backup _Real-time_ product cards.

Jetpack Complete was handled in https://github.com/Automattic/wp-calypso/pull/45009.
Security in https://github.com/Automattic/wp-calypso/pull/45035.

### Implementation notes

For the activity log feature item, I used the copy already used for Jetpack Complete and Security.

### Testing instructions

- Visit the _Plans_ page with the offer reset flow enabled
- Check the copy of the Jetpack Backup product card against the copydeck
- Don't forget to check the descriptions and links in the tooltips
- Click on "Get Backup"
- Check the copy of the Backup _Daily_ and _Real-time_ cards against the copydeck

### Screenshots

Jetpack Backup
<img width="537" alt="Screen Shot 2020-08-19 at 4 48 37 PM" src="https://user-images.githubusercontent.com/1620183/90688498-8e356300-e23c-11ea-907f-2211be007fb2.png">


Backup options
<img width="797" alt="Screen Shot 2020-08-19 at 4 33 42 PM" src="https://user-images.githubusercontent.com/1620183/90688516-91c8ea00-e23c-11ea-8323-44014f3e5ac9.png">
